src/types/items.ts

```diff
@@ -123,13 +123,25 @@ export interface Weapon {
   pbgid?: number;
 }
 
+export interface Ability extends Item {
+  type: "ability";
+  //abilityType?: "always_on" | "timed" | "toggle" | "modal";
+  //activation?: string;
+  activation: "always" | "timed" | "toggle" | "modal";
```

Comment:
Do you know what is the modal type activation? Like vizier points or something?

Reply:
the only modal I have noted is:
abilities/modal_abilities/french/deploy_pavise_fre

it may be something we abstract away from

i cant find vizier points in either places but i probably am overlooking...

```
ls abilities/*/ottoman/*
ls info/buff_info/races/ottoman/ -lh
```

modal abilities are mostly uninteresting to me and there are about 56 files:

```
ls abilities/modal_abilities/*/* -lh
```

Reply:
I like it.  Furthermore, would you like to relate any of the props of abilities like the cooldown to toggle/manual?  There are possibly more properties to go after like if it works for allies or how it targets and how that related to having an aura or not.  Influence based abilities, etc.

I personally don't see vizier as abilities outside of Mehter drums.  If they are handled in techs, it should be sufficient.